src/app/lib/utilities/preprocessor/rules/timestamp.test.js

```diff
@@ -24,6 +24,29 @@ const paragraphBlock = {
   },
 };
 
+const imageBlock = {
```

Comment:
I am not sure what an imageBlock should look like.

Reply:
The Image Container fixture data has helper files for generating an image block https://github.com/bbc/simorgh/blob/latest/src/app/containers/Image/helpers/fixtureData.js#L41
At first glance, though, this might be difficult to read, due to this using functions to generate the JSON. 

So here's an example of an `image` block, containing `rawImage`, `altText` and `caption` blocks: https://github.com/bbc/simorgh/blob/latest/data/prod/news/articles/cyddjz5058wo.json#L34-L108